src/Application/Actions/Person/Create.php

```diff
@@ -135,6 +138,14 @@ protected function action(): Response
         $token = Token::create((string) $person->getId(), false, $person->stripe_customer_id);
         $person->addCompletionJWT($token);
 
+        // After completely persisting Person, send them a registration success email
+        $requestBody = $person->toMailerPayload();
+        $sendSuccessful = $this->mailerClient->sendEmail($requestBody);
+
+        if (!$sendSuccessful) {
+            throw new BadRequestException('Failed to send registration success email to newly registered donor.');
```

Comment:
@alihejazi  I still think this should be a different class if we're going to use one for all failure modes: https://github.com/thebiggive/identity/pull/11#discussion_r969179675

I also don't think we need a new exception class for this if we do think it's more likely to be bad API client input, because Slim already has an exception class for this.